client/compliance_client.go

```diff
@@ -175,7 +169,7 @@ type complianceGrpcClient struct {
 // This service is used to test that GAPICs can transcode proto3 requests to
 // REST format correctly for various types of HTTP annotations.
 func NewComplianceClient(ctx context.Context, opts ...option.ClientOption) (*ComplianceClient, error) {
-	clientOpts := defaultComplianceGrpcCallOptions()
+	clientOpts := defaultComplianceGrpcClientOptions()
```

Comment:
nit: `Grpc` should be `GRPC`. Other examples: [transport.DialGRPC](https://pkg.go.dev/google.golang.org/api/transport#DialGRPC) [option.WithGRPCDialOption](https://pkg.go.dev/google.golang.org/api/option#WithGRPCDialOption)